src/main/java/edu/greenblitz/robotName/subsystems/Arm/ArmMechanism/ArmMechanism.java

```diff
@@ -0,0 +1,65 @@
+package edu.greenblitz.robotName.subsystems.Arm.ArmMechanism;
+
+import edu.greenblitz.robotName.subsystems.Arm.Elbow;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.WristConstants;
+import edu.greenblitz.robotName.subsystems.Arm.Wrist;
+import edu.greenblitz.robotName.utils.GBSubsystem;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import org.littletonrobotics.junction.Logger;
+
+import static edu.greenblitz.robotName.subsystems.Arm.ArmMechanism.ArmMechanismConstants.*;
+import static edu.greenblitz.robotName.subsystems.Arm.ArmMechanism.ArmMechanismConstants.WristMechanismConstants.*;
+import static edu.greenblitz.robotName.subsystems.Arm.ArmMechanism.ArmMechanismConstants.ElbowMechanismConstants.*;
+
+public class ArmMechanism extends GBSubsystem {
+
+    private static Mechanism2d armMechanism;
+
+    private MechanismRoot2d root;
+
+    private final MechanismLigament2d wristJoint;
+
+    private final MechanismLigament2d elbowJoint;
+
+    private final Elbow elbow;
+    private final Wrist wrist;
+
+
+
```

Comment:
No getInstance?

Reply:
If there are no commands for subsystem, is it really a subsystem?

Reply:
i need the periodic and i cant see why not to use it as subsys

Reply:
changed it so the subsystem calls the mechanism periodic